CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## 1.2.0 - 2017-09-11
+### Deprecated
+- The knife-bmcs gem has been deprecated. Users should switch to knife-oci, which provides a similar set of commands under 'knife oci'. knife-oci is a runtime dependency of knife-bmcs, so will automatically be availabe when upgrading.
```

Comment:
may be worth linking to the rename changes file you're working on from here

Reply:
done

Reply:
typo: availabe -> available

Reply:
Good catch, but already removed that sentence.